src/inventory/hosts

```diff
@@ -3,3 +3,59 @@
 # Changes made to this file may be overwritten.
 #
 localhost ansible_connection=local ansible_python_interpreter=python
+
```

Comment:
All of this extra content in hosts is probably left over from a test you ran. This should be reverted out of the hosts file.